validphys2/src/validphys/results.py

```diff
@@ -98,7 +98,7 @@ def central_value(self):
 
     @property
     def std_error(self):
-        return np.sqrt(np.diag(self.covmat))
+        return np.diag(self.sqrtcovmat)
 
```

Comment:
This change doesn't seem connected to the rest. Why the change?

Reply:
Yes, as I wrote in a comment, it is just a bug that I found and corrected. Since it is small I believe we can keep it in this PR but if you prefer I can move it in another PR.

Reply:
if it is a bug better a PR and merge it quickly so it gets fixed (specially given this pr is now a draft)

Reply:
Actually I don't think this is correct. Originally I assumed `self.sqrtcovmat` and `np.sqrt(self.covmat)` would be the same thing, but they are not (as you clearly know). However, for the `std_error` I assume we just want the square root of the variances.

Reply:
> Yes we only want the sqrt of the variances

Then the right order is to first take the diag, then you get the variances, and do the `sqrt` to that:

`np.sqrt(np.diag(self.covmat))`

Reply:
Right but the problem is that self.covmat is not working which is the reason of the bug fix

Reply:
oh, but then we should fix `self.covmat` ? Or that one simply doesn't exist and this was always wrong?

Reply:
(also the two lines are not the same thing, but more than that I want to understand what the bug actually is)

Reply:
I believe the problem is that `self.covmat` is initialized in the class init and that particular class in the code is only used twice. In both cases the covariance matrix is taken from here https://github.com/NNPDF/nnpdf/blob/4170be2630f98420f72037a3099428aed60f2e2e/validphys2/src/validphys/config.py#L838 which I believe, in that case, return the function instead of the object produced by the function (but I don't know why). Note that I discovered this bug trying to produce a report using the function `plot_fancy` which in turn uses https://github.com/NNPDF/nnpdf/blob/4170be2630f98420f72037a3099428aed60f2e2e/validphys2/src/validphys/results.py#L504 which uses again `covariance_matrix`.

Reply:
Apologies, I somehow completely missed your discussion. Could you maybe share the runcard that results in the error?

Reply:
I get no errors when trying [this runcard](https://github.com/NNPDF/nnpdf/blob/master/validphys2/examples/dataspecs.yaml) using this branch but undoing the change at line 101 of `results.py`

Reply:
Mmm then I might have done some mistake myself. Let me revert this commit.